python/cuml/test/dask/test_nearest_neighbors.py

```diff
@@ -88,7 +88,7 @@ def test_compare_skl(nrows, ncols, nclusters, n_parts, n_neighbors,
 
         wait(X_cudf)
 
-        cumlModel = daskNN(verbose=False, n_neighbors=n_neighbors,
+        cumlModel = daskNN(verbose=True, n_neighbors=n_neighbors,
```

Comment:
why this change in this PR? I think it might've slipped in from debugging the dask nn issue?